play-akka-http-ws-standalone/src/main/scala/play/api/libs/ws/akkahttp/StandaloneAkkaHttpWSResponse.scala

```diff
@@ -0,0 +1,82 @@
+package play.api.libs.ws.akkahttp
+
+import akka.http.scaladsl.model.HttpResponse
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.Materializer
+import akka.stream.scaladsl.Source
+import akka.util.ByteString
+import play.api.libs.ws.{ StandaloneWSResponse, WSCookie }
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+private[akkahttp] object StandaloneAkkaHttpWSResponse {
+  def apply(resp: HttpResponse)(implicit mat: Materializer) = new StandaloneAkkaHttpWSResponse(resp)
+}
+
+final class StandaloneAkkaHttpWSResponse private (val response: HttpResponse)(implicit val mat: Materializer) extends StandaloneWSResponse {
+
+  final val UnmarshalTimeout = 1.second
+
+  /**
+   * Return the current headers for this response.
+   */
+  override def headers: Map[String, Seq[String]] =
+    response.headers
+      .groupBy(_.name)
+      .mapValues(_.map(_.value()))
+
+  /**
+   * Get the underlying response object.
+   */
+  override def underlying[T]: T = ???
+
+  /**
+   * The response status code.
+   */
+  override def status: Int = response.status.intValue()
+
+  /**
+   * The response status message.
+   */
+  override def statusText: String = ???
```

Comment:
That's the `reason` field for us : `scala> akka.http.scaladsl.model.StatusCodes.OK.reason`